src/main/kotlin/com/wgslfuzz/analysis/StatementBehaviour.kt

```diff
@@ -187,6 +187,7 @@ private fun statementBehaviour(
 
             // NON-STANDARD: the code here is statically unreachable so has behaviour next.
             is AugmentedStatement.DeadCodeFragment -> setOf(StatementBehaviour.NEXT)
+            is AugmentedStatement.ControlFlowWrapper -> setOf(StatementBehaviour.NEXT)
```

Comment:
Please talk to @JamesLee-Jones about this. I think it might be better to throw an exception if statement behaviour analysis hits an augmented statement (including a `DeadCodeFragment`, as we would need to think through really carefully what would happen there. For James's uniformity project we do not plan to run this on transformed shaders, so an exception would be fine for now.